test/boost/cql_query_test.cc

```diff
@@ -4408,3 +4408,28 @@ SEASTAR_TEST_CASE(test_int_sum_with_cast) {
         });
     });
 }
+
+SEASTAR_TEST_CASE(test_bigint_avg) {
```

Comment:
This test is already passing. Did you intent to use 9223372036854775807 as the constants?

Reply:
yes, probably copy/paste error. good catch.
I'll fix that and resend a new version (including sorting of the error codes).